.github/workflows/check.yml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Check
```

Comment:
The names of other actions seem more descriptive to me. It's redundant (with line 19 and following), but I think it would help to say something like "Check every push to main and pull requests to main" here for readers unfamiliar with the configuration syntax.

Reply:
I see. My assumption was that anyone reading this should be familiar with the syntax.